### PR TITLE
Haskell updates maintainer

### PR DESCRIPTION
**Pull request by thielema**
###### Description of changes

add thielema as maintainer to configuration-hackage2nix/main.yaml

### CODE REVIEW

**Comment by thielema** [comment]
Github wants to pull-request this into master, but I have branched from haskell-updates.

**Comment by maralorn** [comment]
Well, that’s a lot of packages.^^ But I see you maintain all of them on hackage, so we are very happy for you to take ownership for them here, too. Cool.

A couple of points:

1. A while back, I wrote down this information about fixing packages and being a maintainer. https://md.darmstadt.ccc.de/haskell-packages-contribution-checklist#What-does-it-mean-to-be-a-maintainer-of-a-Haskell-package Maybe have a look at that.
2. If a package you maintain is broken by a dependency of it, you will get notified either way, even if you are not a maintainer of a broken dependency. You might consider only becoming a maintainer of some of the packages. But in your case, that doesn’t seem to make a lot of sense.
3. To conform more to the contributor's guide, your commit message should start with a scope. e.g. `haskellPackages: …`
4. You need an entry in `maintainers/maintainer-list.nix`.
5. We, or rather our tooling, only notifies maintainers when packages break. So if you want to confirm that they all work, you will need to check that manually. One idea in this case, with which I would be okay, would be to simply throw them all out of the broken.yaml, merge, let hydra create the report and see how much breakage is happening.

**Comment by maralorn** [comment]
Thank you!

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -15410,6 +15410,12 @@
     githubId = 3268082;
     name = "Thibaut Marty";
   };
+  thielema = {
+    name = "Henning Thielemann";
+    email = "nix@henning-thielemann.de";
+    github = "thielema";
+    githubId = 898989;
+  };
```
Comment (wolfgangwalther) [inline maintainers/maintainer-list.nix]: @thielema it looks like you are not part of the @NixOS/nixpkgs-maintainers team, you likely missed the invite when you became maintainer / the team was created. This means you can't be requested for review properly. Please consider requesting another invite, see https://github.com/NixOS/nixpkgs/tree/master/maintainers#tools-for-maintainers for how to do that.

**Comment by thielema** [comment]
On Thu, 6 Nov 2025, Wolfgang Walther wrote:

> [9132420?s=20&v=4] wolfgangwalther left a comment (NixOS/nixpkgs#227427)
>
>       I rebased my commits from April 2023 and created a new pull-request against haskell-updates. Would this
>       trigger a new invitation?
> 
> Your commits were merged already, there is no point in doing that again. 
> Please read the link I posted earlier, it explains how to get an invite 
> to the team.

That's what I did. It reads:

"How to become a maintainer

...

In order to do so, add yourself to the maintainer-list.nix, and then to 
the desired package's meta.maintainers list, and send a PR with the 
changes."

That's done. Then the doc continues:

"Tools for maintainers

...

New maintainers will automatically get invited to join the 
NixOS/nixpkgs-maintainers GitHub team. By joining, you will get some 
triaging rights in the nixpkgs repository, like the ability to close and 
reopen issues made by others, and managing labels. However, the GitHub 
team invite is usually only sent by email, and is only valid for one week! 
Should it expire, please ask for a re-invite in the GitHub org owners help 
desk Matrix channel."


I have no access to Matrix. How can I retrigger the invitation? Could you 
do this for me?

I guess I cannot simply remove myself as maintainer and re-add, because 
there are already packages maintained by me.

**Pull request merged**